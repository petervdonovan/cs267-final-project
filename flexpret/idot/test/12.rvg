[mu []
[[file {Reset_Handler} {
  Reset_Handler:
    [id init]
    [single t5
      {
        [def (rng) [rng-seeded t4 t5 bits 6 length 6 seeded 2]]
        [iterate {n,rand} of [zip t0 of [.. t3 3 to 9] and t1 of rng]
          {
            [sim-print [fst n,rand] t4]
            [sim-print [snd n,rand] t4]
          }]
      }]
    [id die]
  }]]
]
