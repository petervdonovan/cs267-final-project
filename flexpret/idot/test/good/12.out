Core-0:          3
Core-0:          6
Core-0:          4
Core-0:         26
Core-0:          5
Core-0:         35
Core-0:          6
Core-0:         62
Core-0:          7
Core-0:         32
Core-0:          8
Core-0:          1
Core-0 is done
Core-1 is done
Core-2 is done
Core-3 is done
Core-4 is done
Core-5 is done
Core-6 is done
Core-7 is done
Core-8 is done
Core-9 is done
Core-10 is done
Core-11 is done
Core-12 is done
Core-13 is done
Core-14 is done
Core-15 is done
All cores are done terminating
Assertion failed: Program terminated sucessfully
